packages/clerk-js/src/core/clerk.ts

```diff
@@ -151,7 +151,7 @@ export class Clerk implements ClerkInterface {
   public organization: OrganizationResource | null | undefined;
   public user: UserResource | null | undefined;
   public __internal_country?: string | null;
-  public telemetry?: TelemetryCollector;
+  public telemetry: TelemetryCollector | undefined;
```

Comment:
Updated to `| undefined` instead of `|`, otherwise it leads to a build error: Property 'telemetry' is optional in type 'Clerk' but required in type 'Clerk'.

I think it's also good to keep it consistent with the other attributes in here.